docs/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ nav_order: 5
 
 ## main
 
+* Add visit_rendered_component_in_browser method to enable testing JS interactions in components
```

Comment:
I'll write this here and assume you'll handle the other changes, but I think `with_rendered_component_path` is a good name for this new API ❤️ 

```suggestion
* Add `with_rendered_component_path` method to enable testing components in browser tests.
```